packages/form/addon/components/cf-field/input/radio.hbs

```diff
@@ -22,4 +22,11 @@
       {{option.label}}
     {{/if}}
   </label>
-{{/each}}
\ No newline at end of file
+{{/each}}
+{{#if (and (eq @field.question.raw.isRequired "false") @field.answer.value)}}
+  <br />
+  <a data-test-radio-reset href="#" {{on "click" (fn @onSave null)}}>{{t
```

Comment:
This will cause a `#` being appended to the URL which we don't want. Please add an intermediate action which includes `event.preventDefault()`

Reply:
`done`